src/main/surfrdUtilsMod.F90

```diff
@@ -149,12 +149,10 @@ subroutine collapse_crop_types(wt_cft, fert_cft, cftsize, begg, endg, verbose, s
     ! !DESCRIPTION:
     ! Collapse unused crop types into types used in this run.
     !
-    ! Should only be called if using prognostic crops - otherwise, wt_cft is meaningless
-    !
     ! !USES:
     use clm_varctl , only : irrigate, use_crop
```

Comment:
Had to replace npcropmax (which = 78) with maxveg to make this work for all cases.